### PR TITLE
RATEST-74:Failing Test: Fixing FormTests

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/RATEST-74 cc @k-joseph  @kdaud

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/AddEditDeleteFormTest.java [hunk 1]
```diff
@@ -2,57 +2,79 @@
...
+        if (manageHtmlFormsPage.getElementsIfExisting(By.xpath("//*[contains(text(), '" + name + "')]")).isEmpty()) {
+            manageHtmlFormsPage.clickOnNewHtmlForm();
+            htmlFormsPage.createNewFormTest(name, description, version);
+        }
+        homePage.go();
         homePage.goToManageForm();
         if (!manageForm.addPresent()) {
             manageForm.delete();
         }
+        // add Form
```
Comment (kaweesi) [inline ui-tests/src/test/java/org/openmrs/reference/AddEditDeleteFormTest.java:62]: each add/edit/delete should rather be a separate test method and not all in one

Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/AddEditDeleteFormTest.java:62]: Let me separate them into methods

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/AddEditDeleteFormTest.java:62]: You may also need to rename the class as FormTest

**Comment by sherrif10** [comment]
Hello @kaweesi  @kdaud  , Looking forward to hear from you

File: ui-tests/src/test/java/org/openmrs/reference/UnidentifiedPatientKeyboardTest.java [hunk 2]
```diff
@@ -54,9 +54,9 @@ public void unidentifiedPatientKeyboardTest() throws InterruptedException {
 
         patientDashboardPage = registrationPage.confirmPatient();
         patientDashboardPage.waitForPage();
-
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/UnidentifiedPatientKeyboardTest.java:57]: This has been resolved , trailing spaces

File: ui-tests/src/test/java/org/openmrs/reference/UnidentifiedPatientKeyboardTest.java [hunk 3]
```diff
@@ -54,9 +54,9 @@ public void unidentifiedPatientKeyboardTest() throws InterruptedException {
 
         patientDashboardPage = registrationPage.confirmPatient();
         patientDashboardPage.waitForPage();
-        
+
         patient.uuid = patientDashboardPage.getPatientUuidFromUrl();
-        assertPage(patientDashboardPage.waitForPage());	// remember just-registered patient id, so it can be removed.
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/UnidentifiedPatientKeyboardTest.java:59]: Even this has been catered for

**Pull request closed without merging**